systemd/health-checker.service

```diff
@@ -1,17 +1,16 @@
 [Unit]
 Description=MicroOS Health Checker
 After=cloud-final.service
-After=crio.service
-After=etcd.service
-After=kubelet.service
 After=rebootmgr.service
 After=systemd-logind.service
 Wants=local-fs.target
+Before=boot-complete.target
```

Comment:
IIRC not necessary, as `boot-complete.target` requires this

Reply:
I'll remove it, it was included to mirror the example `/usr/lib/systemd/system/systemd-boot-check-no-failures.service`.